### PR TITLE
163 interactive testing documentation

### PR DESCRIPTION
**Pull request by amullarney**

### CODE REVIEW

**Comment by amullarney** [comment]
Deleted anachronistic step-by-step instruction that no longer apply to InteractiveTesting. Updated ReadMe.

File: InteractiveTesting/models/InteractiveTesting/Components/InteractiveTestbench/TestCases_Nominal/NominalBucket/NominalBucket.xtuml [hunk 1]
```diff
@@ -5,17 +5,15 @@ INSERT INTO O_OBJ
 	'NominalBucket',
 	4,
 	'NominalBucket',
-	'1.	Launch Verifier with InteractiveTesting configuration and logging enabled.
-2.	Execute CarparkControl/Initialization/CreatePEIs
-3.	Execute CarparkControl/Initialization/Run
-4.	Execute InteractiveTestbench/TestCases/NominalBucket/Run
-5.	Verify "bucket successful" in the console log.',
+	'A test bucket which runs a state machine to executes a series of test cases.',
```
Comment (johnrwolfe) [inline InteractiveTesting/models/InteractiveTesting/Components/InteractiveTestbench/TestCases_Nominal/NominalBucket/NominalBucket.xtuml:8]: Typo: "...to executes a..."

File: InteractiveTesting/models/InteractiveTesting/Components/InteractiveTestbench/TestCases_Nominal/UC11_NominalExit/UC11_NominalExit.xtuml [hunk 2]
```diff
@@ -5,18 +5,7 @@ INSERT INTO O_OBJ
...
-4. Execute InteractiveTestbench/TestCases/UC01_NominalEntry/Run().
-5. Wait about 20 seconds and definitely no more than 40 seconds.
-6. Execute InteractiveTestbench/TestCases/UC05_NominalPayment/Run().
-7. Execute InteractiveTestbench/TestCases/UC11_NominalExit/Run().
-8. Using Session Explorer verify expected post conditions.
-Specifically, ensure that the Operator instance in the test bench shows
-true for all boolean attributes, 1 for OccupancyIncreased and
-OccupancyDecreased, and 0 for Occupancy.  Verify that all instances in
-the CarparkControl component are in their "resting" states.',
+	'c',
```
Comment (johnrwolfe) [inline InteractiveTesting/models/InteractiveTesting/Components/InteractiveTestbench/TestCases_Nominal/UC11_NominalExit/UC11_NominalExit.xtuml:8]: I've not loaded this branch and looked at it with the editor, but the 'c' on line 8 concerns me.

**Review by johnrwolfe — CHANGES_REQUESTED** [review]
The additions look great.  Just a couple embedded comments below.

**Pull request merged**